integration_tests/src/main/python/parquet_write_test.py

```diff
@@ -755,3 +755,13 @@ def test_write_with_planned_write_enabled(spark_tmp_path, planned_write_enabled,
         lambda spark, path: spark.read.parquet(path),
         data_path,
         conf)
+
+# Issue to test a known bug https://github.com/NVIDIA/spark-rapids/issues/8694 to avoid regression
+@ignore_order
+def test_write_list_struct_single_element(spark_tmp_path):
```

Comment:
Either there should be a similar test in parquet_test to test the GPU read side or this should be extended to test the read on the GPU.  This is only testing GPU write, but the CPU performs both reads.  To add the test of the GPU read here, could add something like:
```
  cpu_path = data_path + "/CPU"
  assert_gpu_and_cpu_are_equal_collect(lambda spark: spark.read.parquet(cpu_path), conf)
```